e-commerce-frontend/src/app/seller/seller.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../_services/user.service';
+
+@Component({
+  selector: 'app-seller',
+  templateUrl: './seller.component.html',
+  styleUrl: './seller.component.css'
+})
+export class SellerComponent implements OnInit {
+
+  message: string = '';
+  constructor(private userService: UserService) { }
+  ngOnInit(): void {
+    this.forSeller();
+  }
```

Comment:
Try to follow a single type of code formatting and indentation. In some files you have:

variable
[space]
constructor
[space]
ngOnInit()
[space]
In other classes, you have:
variable
constructor
ngOnInit()
[space]